### ISSUE TITLE
Adding Multiple Issues

### ISSUE THREAD

**Issue by TheAdvertisement** [piskelapp/piskel#959]
There have been many times where I've wanted to import multiple frames of a sprite into piskel, however this has to be done one at a time. You can just drag a sprite off of the window, but even if it's the same size it will only go onto the frame you're currently on, and will stack on each other on the same layer (if it's not the same size, the usual prompt will come up asking if you want to change the canvas size or no, and where you want to import the sprite to, which isn't bad, though I feel there could be a quicker way to do this). Importing the sprites one by one can get very tedious very quickly. 
There is a way around this, by compacting the sprites into a spritesheet, so you only import one image, but by doing this you lose the ability to see the frames lined up, so you can look at the animation while editing.
I propose that, if the sprite you're importing is the same size as the canvas, you should be able to just drag it off the window, and it will be added onto the same layer that you're currently on, and onto a new frame right after the one you're on.

**Comment by OPStellar** [comment]
Yes, it does work similarly to your proposition, but instead of adding the imported image in a new frame, what it has done for me is place down where my cursor is when I drop it. I, too would like that to change to adding to a new frame.
Another action it does currently when importing multiple sprite files from Finder/Explorer is exactly as you described; it places everything center-frame on new frames, in the sequence by which they were selected or sorted.

I would like to be able to import multiple sprite files at once without dragging, though, even if it was limited to only sprites that are equal in dimensional size.

**Comment by TheAdvertisement** [comment]
I haven't been able to import multiple files without dragging. The only way for me to do so would be to highlight all the images I want in the file explorer and drag them on, but it doesn't allow me to highlight multiple. I'm pretty sure this is handled by the website, because on other sites I can do so. I use google chrome on Windows 10, if that makes any difference.

**Comment by OPStellar** [comment]
I use the desktop app and it seems to function the same way. I cannot highlight multiple if I am importing within the app, but I can if I am dropping from outside the app, though in those cases it does not prompt me about anything; it just imports them at the size the canvas already is.

**Comment by TheAdvertisement** [comment]
I tested it out, and it works pretty much perfectly to how I could've wanted it! You do have to import the first sprite and set the canvas to the right size first, but then you can add the rest of the sprites in the rest of the frames!
Slightly annoying that I can't use the import option and have to go out of the way to my file explorer, but it's not that big a difference and is the perfect alternative.
I'd still like it to work when using the import option directly, and it seems it won't be too hard since that functionality already exists, but it's not really urgent anymore. Thanks for the help!
